package.json

```diff
@@ -121,5 +121,8 @@
   },
   "engines": {
     "node": ">=14"
+  },
+  "devDependencies": {
+    "isbot": "^3.5.1"
```

Comment:
all dependencies in the workspace root are "dev dependencies" so we just put them all in "dependencies"